locale/fi/LC_MESSAGES/django.po

```diff
@@ -3574,6 +3579,74 @@ msgstr "Ei osumia"
 msgid "Search for..."
 msgstr "Etsi..."
 
+#: templates/support_channels.html
+msgid "Documentation"
+msgstr "Asiakirjat"
+
+#: templates/support_channels.html
+msgid "Aplus manual"
+msgstr "Aplus-käsikirja"
+
+#: templates/support_channels.html
+msgid "Aplus website"
+msgstr "Aplus-verkkosivusto"
+
+#: templates/support_channels.html
+msgid "Community"
+msgstr "Yhteisö"
+
+#: templates/support_channels.html
+msgid "Microsoft Teams"
+msgstr "Microsoft Teams"
+
+#: templates/support_channels.html
+msgid "Create an issue in Github"
+msgstr "Luo ongelma Githubissa"
+
+#: templates/support_channels.html
+msgid "Announcements"
+msgstr "Ilmoitukset"
+
+#: templates/support_channels.html
+msgid "Current Development"
+msgstr "Nykyinen kehitys"
+
+#: templates/support_channels.html
+msgid "Other servers that use A+"
+msgstr "Muut A + -palvelinta käyttävät palvelimet"
```

Comment:
"Muut A + -palvelinta käyttävät palvelimet" change to "Muut {brand}-alustaa käyttävät palvelimet"